dvc/command/experiments/show.py

```diff
@@ -473,7 +383,22 @@ def show_experiments(
         td.drop_duplicates("cols", ignore_empty=False)
 
     if pcp:
-        td.dropna("rows", how="all")
+        if drop is None:
```

Comment:
For myself, I would prefer first to do the regex match and then add `Created` to the drop_set.

Reply:
Done